ampersand-multifield-view.js

```diff
@@ -39,19 +33,17 @@ var MultiFieldView = View.extend({
       throw new Error('must pass in a name');
     }
 
-    if (!this.fields) {
-      throw new Error('must have an array of fields');
-    }
-
-    this.value = spec.value || {};
+    this.beforeSubmit = spec.beforeSubmit || this.beforeSubmit;
+    this.fields = spec.fields || this.fields || [];
```

Comment:
on a State object, if you set a property that isn't defined in the `props` it gets ignored by default. doesn't look like you're actually using `props`, so it doesn't matter. carry on!